### PR TITLE
New version: Joplin.Joplin version 2.13.13

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133295)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-08T20:03:35.0356855Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 536.)

File: manifests/j/Joplin/Joplin/2.13.13/Joplin.Joplin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- interactive
+- silent
+InstallerSwitches:
+  Custom: /CURRENTUSER /RESTARTAPPLICATIONS
+UpgradeBehavior: install
+ReleaseDate: 2024-01-06
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/laurent22/joplin/releases/download/v2.13.13/Joplin-Setup-2.13.13.exe
+  InstallerSha256: 8310BB2B4F4020BC1CDB637D8317B0824D3DC5886FA15F555A3FA266A5030390
```
Comment (stephengillie) [inline manifests/j/Joplin/Joplin/2.13.13/Joplin.Joplin.installer.yaml:21]:
```suggestion
```

**Review by stephengillie — APPROVED** [review]

**Comment by Exorcism0666** [comment]
??

**Comment by Exorcism0666** [comment]
@stephengillie idk what's going on haha

**Comment by stephengillie** [comment]
> @ stephengillie idk what's going on haha

Very sorry - our bots loop like this sometimes. I try to catch these at the start of my day.

**Comment by futal** [comment]
Hi @Exorcism0666.
I noticed that your push request has included only the "user" scope but not the "machine" scope. It means that "winget update joplin" updates to version 2.13.13 if installed for a single user but only to version 2.13.12 if installed machine wide. 

Could you add the "machine" scope to `manifests/j/Joplin/Joplin/2.13.13/Joplin.Joplin.installer.yaml`? See `manifests/j/Joplin/Joplin/2.13.12/Joplin.Joplin.installer.yaml` to have two scopes.

Thanks a lot for maintening Joplin entry in this repository, it is so convenient to be able to update with Winget.

**Pull request merged**